src/openforms/registrations/contrib/email/utils.py

```diff
@@ -0,0 +1,33 @@
+from openforms.config.models import GlobalConfiguration
+
+
+def get_registration_email_templates(submission):
+    config = GlobalConfiguration.get_solo()
+
+    subject_template = (
+        submission.form.registration_email_subject or config.registration_email_subject
+    )
+    payment_subject_template = (
+        submission.form.registration_email_payment_subject
+        or config.registration_email_payment_subject
+    )
+
+    if any(
+        [
+            submission.form.registration_email_content_html,
+            submission.form.registration_email_content_text,
+        ]
+    ):
+        return (
+            subject_template,
+            payment_subject_template,
+            submission.form.registration_email_content_html,
+            submission.form.registration_email_content_text,
+        )
+
+    return (
+        subject_template,
+        payment_subject_template,
+        config.registration_email_content_html,
+        config.registration_email_content_text,
+    )
```

Comment:
Maybe define an explicit return type instead of a generic 4-tuple. It's clearer and more flexible to change. A simple `namedtuple` or `dataclass` will do. I think Sergei prefers the latter.

A 3-tuple, like from `EmailRegistration.render_registration_email`, is the borderline of my taste.